src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -407,13 +441,23 @@ void ClusterResourceScheduler::DeleteResource(const std::string &node_id_string,
   };
   if (idx != -1) {
     it->second.predefined_resources[idx].total = 0;
+
+    if (node_id == local_node_id_) {
+      local_resources_.predefined_resources[idx].total.clear();
```

Comment:
Is it a bug you found? This code block makes sense, but it doesn't seem to be related to this PR.

Reply:
Yeah you can either think of it as a bug, or out of scope of what the function used to do. Either way, it's intention and necessary.